grepos/app/src/main/java/net/boondockradio/grepos/MainActivity.java

```diff
@@ -74,25 +74,38 @@ public void onScrolled(RecyclerView recyclerView, int dx, int dy) {
 
     private void fetchRepositories() {
         GithubApi api = ApiClient.getClient().create(GithubApi.class);
-        Call<Repositories> call = api.getRepositories("language:java", "Repositories", ++mPage + "", PER_PAGE);
 
-        call.enqueue(new Callback<Repositories>() {
-            @Override
-            public void onResponse(Call<Repositories> call, Response<Repositories> response) {
-                Log.d(TAG, "success");
-                mProgress.setVisibility(View.GONE);
-
-                mAdapter.add(response.body().items);
-                mAdapter.setIsLoading(false);
-                isLoading = false;
-                mTotal = mAdapter.getItemCount();
-            }
+        api
+                .getRepositories(
+                        "language:java",
+                        "Repositories",
+                        ++mPage + "",
+                        PER_PAGE
+                )
+                .subscribeOn(Schedulers.io())
+                .observeOn(AndroidSchedulers.mainThread())
+                .subscribe(new Observer<Repositories>() {
```

Comment:
When you subscribe, you MUST unsubscribe. (unless what you subscribe is synchronous work) 

See below:
- https://github.com/ReactiveX/RxJava
- http://qiita.com/yuya_presto/items/c8c3d77ac958c9c8f67b